### PR TITLE
Issue#44 making timeline posts

### PR DESCRIPTION
**Pull request by AlpErdinc**
Added user ability to make a post.
Post is saved to database under postsMade.
Posts contain user ID, Firstname, Lastname, description (content of the post) and a timestamp.

### CODE REVIEW

File: src/server/controllers/usersController.js [hunk 1]
```diff
@@ -263,6 +263,29 @@ const addJobAppliedToUser = async(req, res) => {
...
+const addTimelinePost = async(req, res) => {
+    const { _id, firstname, lastname, description, timestamp} = req.body;
+    const user = await User.findByIdAndUpdate( _id , {
+        $addToSet: {
+            postsMade: { _id: _id, firstname:  firstname , lastname: lastname, description: description, timestamp: timestamp },
+        },
+    });
+    if (user) {
+        console.log("Succesfully updated awaiting connections");
+        res.status(200).json({ message: "Succesfully added user `${_id}`" });
```
Comment (Resident-png) [inline src/server/controllers/usersController.js:277]: For this message log, I would maybe change it to "post from (id) saved successfully".

**Review by Resident-png — APPROVED** [review]
Overall, this seems good. Would have to view this in my browser to give further comments on the ui.

**Review by imran1289-ah — APPROVED** [review]
everything looks good for me

File: src/client/src/components/UserTimeline.jsx [hunk 2]
```diff
@@ -68,14 +107,21 @@ const UserTimeline = () => {
...
                 variant="standard"
                 className="postTextField"
+                value= {postData.description}
+                onChange={e => setpostData({ ...postData, description: e.target.value })}
               />
+              <div className="timestamp" value={Date.now()} 
+              onChange={e => setpostData({...postData, timestamp: e.target.value})} />
+                
+              {/* <input className="imageFile" type="file" accept=".png" name="image" 
+              onChange={e => setpostData({...postData, attachment: e.target.files[0]})}/> */}
```
Comment (Aboudik1452) [inline src/client/src/components/UserTimeline.jsx:124]: This commented section is for attaching an image to the post, right?

**Review by Aboudik1452 — APPROVED** [review]

**Pull request merged**